styli.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-link="https://source.unsplash.com/random/"
+link="https://source.unsplash.com/random/1920x1080"
```

Comment:
This breaks [line 182](https://github.com/thevinter/styli.sh/blob/232f23ccb0dc9701372bc8aafd5b67399917c51e/styli.sh#L182)

Reply:
I missed that but, it should be ok now